src/test/scala/com/cognite/spark/datasource/DataPointsRelationTest.scala

```diff
@@ -241,6 +241,7 @@ class DataPointsRelationTest extends FlatSpec with Matchers with SparkTest {
       .load()
     destinationDf.createTempView("destinationDatapoints")
 
+    spark.sparkContext.setLogLevel("OFF")
```

Comment:
I understand why right now but we should add a comment and explain the why, because future developers (possibly including me!) will not understand this. :)

Reply:
Good point